.github/workflows/copilot-review.yml

```
@@ -0,0 +1,183 @@
+            - Hardcoded secrets or API keys
+            - ESLint rule violations
+            - Missing tests for new PR automation features
+            - Improper handling of sensitive PR data
+
+            Provide constructive, actionable feedback with specific code examples.
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+
+      - name: 🔍 Validate PR Against Build Rules
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2162:info:8:45: read without -r will mangle backslashes

(shellcheck)